### DISCUSSION TITLE
How to visualize MobiDS-Ontology.ttl

### DISCUSSION THREAD

**Discussion by thilo20** [Mobility-Data-Space/mobility-data-space#3] [Q&A]
Hi guys,

for https://github.com/Mobility-Data-Space/mobility-data-space/blob/main/ontology/MobiDS-Ontology.ttl
is there a viewer you can recommend to visualize the ontology?

I used [WebVOWL](http://www.visualdataweb.de/webvowl/)
http://www.visualdataweb.de/webvowl/#opts=filter_disjoint=false;#file=MobiDS-Ontology.ttl
to visualize and create this screenshot
![MobiDS-Ontology](https://user-images.githubusercontent.com/17619950/126281240-5b73ceb0-7734-44fe-8530-2e888cfffb99.png)

These link might also help with the format:
- https://de.wikipedia.org/wiki/Turtle_(Syntax)
- https://de.wikipedia.org/wiki/Resource_Description_Framework

**Comment by JohannesLipp** [comment]
Hi @thilo20 we used the same Page (WebVOWL) in other projects before. 

For manual visualizations, your progress is totally fine. One hint: Instead of taking screenshots, that website also supports an export feature that you can use to produce SVG images instead of PNGs. They provide better quality.

I have also scripts available that automatically generate a fully-fletched ontology documentation page including that visualization. Example outputs are [Gaia-X](https://w3id.org/gaia-x/core) or the [IDS](https://w3id.org/idsa/core). They are integrated into GitLab or GitHub pipelines that are triggered automatically to keep the output always up-to-date.
